### PR TITLE
Edit c sharp comments.md

### PR DESCRIPTION
**Pull request by scardonacervantes**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by scardonacervantes** [comment]
Did not touch XML comments section, as I believe they should have a separate entry.

File: content/c-sharp/concepts/comments/comments.md [hunk 1]
```diff
@@ -13,46 +13,73 @@ CatalogContent:
...
+This is a
+multi-line comment
 */
 ```
+Multi-line comments can also be inserted within a line of code, allowing for additional executable code after the comment, as they are explicitly closed with `*/`.
 
-### Example
+```cs
+int number = 10; /* Multi-line comment */ int anotherNumber = 20;
+```
```
Comment (PragatiVerma18) [inline content/c-sharp/concepts/comments/comments.md:42]: This is not really required, the above explanation already covers this. We should remove it.

File: content/c-sharp/concepts/comments/comments.md [hunk 2]
```diff
@@ -13,46 +13,73 @@ CatalogContent:
...
+            */
+
+            int number = 10; /* This is a multi-line comment inserted within a line of code */ int anotherNumber = 20;
+
+            // This is another single-line comment
+            Console.WriteLine("The number is: " + number);
+            Console.WriteLine("Another number is: " + anotherNumber);
+        }
+    }
+}
```
Comment (PragatiVerma18) [inline content/c-sharp/concepts/comments/comments.md:75]: This example is too long, we should keep it simple as the existing example.

File: content/c-sharp/concepts/comments/comments.md [hunk 3]
```diff
@@ -13,46 +13,73 @@ CatalogContent:
...
 ```
 
-### XML Comments
+## XML Comments
 
 [XML](https://www.codecademy.com/resources/docs/general/xml) comments are structured comments that produce [API](https://www.codecademy.com/resources/docs/general/api) documentation. The C# compiler produces an XML file that contains structured data representing the comments. Other tools can process that XML output to create human-readable documentation in the form of web pages or PDF files, for example.
 
-#### Syntax
+### Syntax
```
Comment (PragatiVerma18) [inline content/c-sharp/concepts/comments/comments.md:82]: Syntax should be kept as H2

```suggestion
## Syntax
```

File: content/c-sharp/concepts/comments/comments.md [hunk 4]
```diff
@@ -68,7 +95,7 @@ XML Comments go here
 */
 ```
 
-#### Example
+### Example
```
Comment (PragatiVerma18) [inline content/c-sharp/concepts/comments/comments.md:98]: Example should also be kept as H2

```suggestion
## Example
```

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]
Hey @scardonacervantes, thanks for the contribution, I have some suggestions. Please check and do the required changes. Let me know if you have any questions.

**Comment by PragatiVerma18** [comment]
Hi @scardonacervantes, Thank you for your contribution! While we appreciate your effort, we aim to keep the docs concise, and some of these changes may not align with that goal. 

For now, I’ll close this PR, but we’d love to see more relevant suggestions in the future. Happy contributing!

**Pull request closed without merging**